src/app/base/components/node/NodeTestDetails/NodeTestDetails.test.tsx

```diff
@@ -231,19 +194,14 @@ describe("NodeTestDetails", () => {
 
     const store = mockStore(state);
 
-    mount(
-      <Provider store={store}>
-        <MemoryRouter initialEntries={["/machine/abc123/testing/1/details"]}>
-          <CompatRouter>
-            <Routes>
-              <Route
-                element={<NodeTestDetails getReturnPath={getReturnPath} />}
-                path="/machine/:id/testing/:scriptResultId/details"
-              />
-            </Routes>
-          </CompatRouter>
-        </MemoryRouter>
-      </Provider>
+    renderWithBrowserRouter(
+      <Routes>
+        <Route
+          element={<NodeTestDetails getReturnPath={getReturnPath} />}
+          path="/machine/:id/testing/:scriptResultId/details"
+        />
+      </Routes>,
+      { route: "/machine/abc123/testing/1/details", store }
```

Comment:
You should be able to use `routePattern` instead.
```suggestion
    renderWithBrowserRouter(<NodeTestDetails getReturnPath={getReturnPath} />, {
      route: "/machine/abc123/testing/1/details",
      routePattern: "/machine/:id/testing/:scriptResultId/details",
      store,
    });
```